package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@department-of-veterans-affairs/component-library",
-  "version": "2.1.0",
+  "version": "2.4.0",
```

Comment:
If you want to merge all of these analytics PRs in around and publish them at the same time, you'll only need to increment the version once.